GNUmakefile

```diff
@@ -43,5 +44,9 @@ test-compile:
 	fi
 	go test -c $(TEST) $(TESTARGS)
 
-.PHONY: build test testacc vet fmt fmtcheck errcheck vendor-status test-compile
+test-serv: fmtcheck
+	@docker pull "consul:$(CONSUL_VERSION)"
+	docker run -p 127.0.0.1:8500:8500 "consul:$(CONSUL_VERSION)"
```

Comment:
Should we add `--rm` here to delete the test container after it exits?

Reply:
Yeah good call, added that. Thanks!